src/storage.rs

```diff
@@ -180,49 +194,77 @@ pub trait Storage {
     fn block_hash(&self, number: &u64) -> Result<B256, Self::Error>;
 }
 
+/// Errors when wrapping storage.
```

Comment:
`/// revm [Database] errors when using pevm [Storage] as the underlying provider.`